pyworkforce/staffing/multi_zone_planner.py

```diff
@@ -383,25 +384,16 @@ def roster(self):
 
             shift_names = shifts_info["shifts"]
             shifts_hours = [int(i.split('_')[1]) for i in shifts_info["shifts"]]
```

Comment:
here we shall take into account _working hours_
In shifts info we store _total_ shift time. in reality, when we compare 176 (min/max) it is assumed for _working_ hours:
e.g. in 9hr shift, there are 3 unpaid breaks, hence working hours = 8